azure_storage_client.go

```diff
@@ -335,3 +336,11 @@ func (util *snowflakeAzureClient) detectAzureTokenExpireError(resp *http.Respons
 	return strings.Contains(errStr, "Signature not valid in the specified time frame") ||
 		strings.Contains(errStr, "Server failed to authenticate the request")
 }
+
+func createContainerClient(clientUrl string) (*container.Client, error) {
+	return container.NewClientWithNoCredential(clientUrl, &container.ClientOptions{ClientOptions: azcore.ClientOptions{
```

Comment:
Do we have a way to use authenticated proxy with user and password requirement in go driver?